src/viewer/reference/native_plugins.md

```diff
@@ -134,6 +134,7 @@ This plugin display the 3D representation of the IFC.
 
 | Name                | Payload                                                                                                                            | Description                                   | Emitted on                         |
 | :------------------ | :--------------------------------------------------------------------------------------------------------------------------------- | :-------------------------------------------- | :--------------------------------- |
+| `3d-model-loading`  | `{ ifc, plugin }`                                                                                                                  | Emitted when a 3D model is called.            | `localContext` and `globalContext` |
```

Comment:
I'm not sure that "**called**" is the right verb here.
What do you think about: "*Emitted when a 3D model start loading.*" ?